### PR TITLE
Release 0.14 preparations

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: docs/_templates/topbar/launchbuttons.html [hunk 1]
```diff
@@ -30,21 +33,6 @@
...
-                class="btn btn-secondary topbarbtn">v0.6</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.5.0"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.5</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.4.0"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.4</button></a>
-        <a class="dropdown-buttons"
-            href="https://scipp.github.io/release/0.3.0"><button type="button"
-                class="btn btn-secondary topbarbtn">v0.3</button></a>
```
Comment (jl-wynen) [inline docs/_templates/topbar/launchbuttons.html:47]: Why did you remove these?

Comment (SimonHeybrock) [reply docs/_templates/topbar/launchbuttons.html:47]: I felt the menu was getting too long to be useful. No one is using these anymore, so other than support a historian I see little use?

Comment (jl-wynen) [reply docs/_templates/topbar/launchbuttons.html:47]: Then what is the point of keeping the actual pages around? Should we decide generally on when we remove links / pages?

Comment (SimonHeybrock) [reply docs/_templates/topbar/launchbuttons.html:47]: - Yes we should probably (at some point) also remove the pages. They take a lot of space.
- I don't think if we can determine a fixed cycle at this point. "3 years" might work once we are quite stable, but now that is useless.

**Review by jl-wynen — APPROVED** [review]
